docs/stability.rst

```diff
@@ -98,10 +98,10 @@ The current planned and existing sub-packages are:
                 astropy.constants
             </td>
             <td align='center'>
-                <span class="stable"></span>
+                <span class="mature"></span>
             </td>
             <td>
-                Constants were changed to <tt class="docutils literal"><span class="pre">Quantity</span></tt> objects in v0.2. Since then on, the package has been stable, with occasional additions of new constants.
+                Constants were changed to <tt class="docutils literal"><span class="pre">Quantity</span></tt> objects in v0.2. Since then, the package has been stable, with occasional additions of new constants.
```

Comment:
I don't think this has to have historical completeness. How about:
```
The package has been stable except for the occasional additions of new constants.  Since v3.0(???), it includes the ability to use sets of constants from previous versions.
```